test/auto/injectorSpec.js

```diff
@@ -302,6 +302,26 @@ describe('injector', function() {
           expect(instance.aVal()).toEqual('a-value');
         });
 
+        if (/chrome/.test(navigator.userAgent)) {
```

Comment:
The situation in tests is a little different; in jQuery e.g. we don't sniff in source but if a specific bug exists in a specific browser & version we do sniff in tests. This is not only because it's simpler to do but also to rule out the possibility that a broken support check will make the tests be skipped - we do want to verify that in these specific browsers they're being run.

I agree with you here, though as we're talking about a bleeding-edge feature that browsers are only implementing now and more & more of them will have it in a working state. That's different.